virttest/utils_misc.py

```diff
@@ -4045,26 +4047,39 @@ def get_model_features(model_name):
     """
     features = []
     conf = "/usr/share/libvirt/cpu_map.xml"
+    conf_dir = "/usr/share/libvirt/cpu_map/"
 
     try:
-        output = open(conf, 'r').read()
-        root = ET.fromstring(output)
-        # Find model
-        for model_n in root.findall('arch/model'):
-            if model_n.get('name') == model_name:
-                model_node = model_n
-                for feature in model_n.findall('feature'):
-                    features.append(feature.get('name'))
-                break
-        # Handle nested model
-        nested_model = model_node.find('model')
-        if nested_model is not None:
-            nested_model_name = nested_model.get('name')
-            for model_n in root.findall('arch/model'):
-                if model_n.get('name') == nested_model_name:
-                    for feature in model_n.findall('feature'):
-                        features.append(feature.get('name'))
-                    break
+        if not libvirt_version.version_compare(5, 0, 0):
+            if os.path.exists(conf):
```

Comment:
For compare version, no need to judge the file exist or not. If not exist, it should be error. Also the following part.